Modules/Loadable/Markups/MRML/vtkMRMLMarkupsLineNode.h

```diff
@@ -67,6 +67,9 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsLineNode : public vtk
   void operator=(const vtkMRMLMarkupsLineNode&);
 
   void UpdateMeasurements() override;
+
+  /// Calculates the handle to world matrix based on the current control points
+  void UpdateInteractionHandleModelToWorld() override;
```

Comment:
`UpdateInteractionHandleModelToWorld` -> `UpdateInteractionHandleMatrixToWorld` ?

Reply:
Done!

Reply:
Sorry, but if we want to be consistent then we need to put "Matrix" to the end of the name: UpdateInteractionHandleToWorldMatrix. This is not something that I would delay the pull request for, so only change it if you change other things anyway.

Reply:
Ok, changed to UpdateInteractionHandleToWorldMatrix.